src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export function Person(person) {
```

Comment:
it's only person, because you have one prop `person`. it's incorrect to call it person. but you can destructure and get the person prop

```suggestion
export function Person({ person }) {
```